build.gradle

```diff
@@ -16,13 +16,14 @@ if (!hasProperty('mainClass')) {
 repositories {
     mavenLocal()
     jcenter {
-        url "http://jcenter.bintray.com/"
```

Comment:
This removes java 7 compat, while this is probably fine, you could also write it as `jcenter()` then instead.

Reply:
Change to `jcenter()`